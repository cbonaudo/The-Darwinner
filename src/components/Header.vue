<template>
  <div class="header">
    <h1>ThE DaRwInNeRâ„¢</h1>
    <p class="subtitle">
      <span>"Evolve this"</span> -
      <span>Version 0.1.0</span> -
      <span>cbonaudo</span>
    </p>
  </div>
</template>

<script>
export default {
  name: "Header"
};
</script>

<style>
.header {
  margin-bottom: 50px;
}
.subtitle {
  margin-top: -20px;
}
</style>
