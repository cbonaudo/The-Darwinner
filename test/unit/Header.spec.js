import Header from '@/components/Header'
import { shallowMount } from '@vue/test-utils'

describe('Header.vue', () => {
  const wrapper = shallowMount(Header)

  it('should render correct contents', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
