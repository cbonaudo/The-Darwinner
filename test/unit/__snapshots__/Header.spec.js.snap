// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue should render correct contents 1`] = `
<div
  class="header"
>
  <h1>
    ThE DaRwInNeRâ„¢
  </h1>
   
  <p
    class="subtitle"
  >
    <span>
      'Evolve this'
    </span>
     -
    
    <span>
      Version 0.2.0
    </span>
     -
    
    <span>
      github.com/cbonaudo
    </span>
  </p>
</div>
`;
